.eslintrc.js

```diff
@@ -1,3 +1,3 @@
 'use strict';
 
-module.exports = require('pa11y-lint-config/eslint/es6');
+module.exports = require('pa11y-lint-config/eslint/es2017');
```

Comment:
`pa11y-lint-config` should be updated to point to `1.2.0` in `package.json` as well, otherwise people installing modules from a cached source might not get the version that includes `es2017`